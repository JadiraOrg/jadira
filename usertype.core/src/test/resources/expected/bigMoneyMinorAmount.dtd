<!ELEMENT dataset (
    BIGMONEYMINORAMOUNT*)>

<!ELEMENT BIGMONEYMINORAMOUNT EMPTY>
<!ATTLIST BIGMONEYMINORAMOUNT
    ID CDATA #REQUIRED
    BIGMONEY CDATA #IMPLIED
    NAME CDATA #IMPLIED
>

