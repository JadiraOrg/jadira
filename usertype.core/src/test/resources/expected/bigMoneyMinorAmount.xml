<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE dataset SYSTEM "bigMoneyMinorAmount.dtd">
<dataset>
  <bigMoneyMinorAmount ID="0" BIGMONEY="10000" NAME="test_0"/>
  <bigMoneyMinorAmount ID="1" BIGMONEY="10010" NAME="test_1"/>
  <bigMoneyMinorAmount ID="2" BIGMONEY="99" NAME="test_2"/>
  <bigMoneyMinorAmount ID="3" NAME="test_3"/>
</dataset>